brave-snouts-frontend/src/app/shared/support/support.component.html

```diff
@@ -0,0 +1,3 @@
+<h2>KONTAKT EMAIL</h2>
+<br>
+<p>support.hrabre-njuske@gmail.com</p>
```

Comment:
Add close button,

you can search template on similar dialogs just search "dialog" over project and find HTML that contains close button

In that dialog i'd like to just put in email in the center that's it no H2 tags and similar